addons/website/views/snippets/s_masonry_block.xml

```diff
@@ -309,6 +291,23 @@
             </we-select>
         </div>
     </xpath>
+
+    <!--  Box Vertical Alignment -->
+    <xpath expr="//div[@data-js='WebsiteAnimate']" position="after">
+        <div data-selector=".s_masonry_block .o_grid_item">
```

Comment:
This would also alow to use this feature for `o_grid_item_image`.
But this won't provide any change since the image has `height: 100% !important`.
![Capture d’écran 2024-07-19 à 16 57 46](https://github.com/user-attachments/assets/46b15abf-193a-4ec1-8c75-aea8f1df62e3)

To be tested, but I think we could avoid theses image blocks :
```suggestion
        <div data-selector=".s_masonry_block .o_grid_item" data-exclude=".o_grid_item_image">
```

Reply:
God catch, thanks 👍